info_tools/get_status.py

```diff
@@ -0,0 +1,557 @@
+#!/usr/bin/env python2.7
+
+# !! TODO - Argparse
```

Comment:
yes, this either needs `argparse`, some sort of help flag, or minimally, if the number of arguments is not what was expected it should output what is expected for arguments.  currently it either outputs nothing or `None`, which is not helpful.